index.html

```diff
@@ -12220,14 +12228,55 @@ <h3>Examples</h3>
 
 <section id="security" class="appendix">
   <h3>Security Considerations</h3>
-  <p class="ednote">Consider requirements from <a href="https://w3ctag.github.io/security-questionnaire/">Self-Review Questionnaire: Security and Privacy</a>.</p>
-  <p>See, <a href="#iana-considerations" class="sectionRef"></a></p>
+  <p>See, <a href="#iana-security">Security Considerations</a> in <a href="#iana-considerations" class="sectionRef"></a>.</p>
```

Comment:
(I have not checked before, sorry): I guess these sections (security, privacy, etc) should not be appendices, but are supposed to be sections in the final text...